docs/requirements/requirements.md

```diff
@@ -0,0 +1,310 @@
+# Requirements
+
+:::caution
+
+This document is in the making since we are still in the requirements engineering phase.
+
+:::
+
+## RE Wording in this Document
+
+- We use *Microservice* and *Service* interchangeably.
+- We use *Reference Architecture* and *System* interchangeably.
+- RA = reference architecture
+
+## Stakeholders
+
+- Prof. Dr. Steffen Becker -- The Examiner
+- Sandro Speth -- Supervisor
+- Niklas Meißner -- Supervisor
+- The Developers
+- S -- Researcher
+- F -- Researcher
+- SH -- Researcher
+
+## Requirement Categories
+
+- Constraints
+- Process
+  - EnPro -- The project itself
+  - Software Artefacts
+- Product
+  - Domain
+  - Usability
+  - Non-functional Requirements & Quality Attributes
+  - Performance
+- Design
+  - Architecture
+  - Microservice Patterns
+  - Loose Coupling
+  - Technologies
+  - Deployment
+
+## Constraints
+
+### R-C-001: No external services
+
+- We are not allowed to use external services (e.g., external microservices).
```

Comment:
I would prefer if we write it from the system's perspective and not from what we are allowed to do.
Imho regarding requirements, the system should be central

Reply:
I agree. I have changed that, see the latest commit.